drf_jsonapi/exception_handlers.py

```diff
@@ -70,16 +65,13 @@ def handle_Error(cls, exc, context):  # NOSONAR
         :return: A 400 error Response
         :rtype: rest_framework.response.Response
         """
-
+        del context
```

Comment:
I was waiting for someone to ask about this. I'm not crazy about it.

So here's the issue. I remove `context` from the method I get a linter error that I've changed the method signature. If I leave it in I get a linter error that I've got an unused parameter. If I add `# noqa` it's overkill.

So I did some searching and found this: https://stackoverflow.com/questions/10025680/how-can-i-denote-unused-function-arguments

If you have any ideas I am open to doing something better!

Reply:
Very interesting. Given the problem you're trying to solve I think this approach is fine. Maybe just add a comment about your intention though since it is not clear on its face.

Reply:
Looking into this closer I found out I can prefix the parameter with `_` to indicate that it's not used. PyLint is happy with that.